github-management/org-owners-guide.md

```diff
@@ -41,7 +41,8 @@ managed by the Kubernetes project or use a different name.
 
 Due to licensing and CLA issues, prior to transferring software into a
 Kubernetes managed organization there is some due diligence that needs to occur.
-Please contact the steering committee and CNCF prior to moving any code in.
+If needed, please contact the steering committee and CNCF prior to moving any
```

Comment:
We don't need to contact the steering committee + CNCF everytime; we use FOSSA license scan now.

I'll add instructions/process for migrating repos in a follow-up PR, want to discuss something first.

Reply:
The "if needed" is a little vague here, when would I need to? But we can hash that out in the followup